### PR TITLE
Add sample pdf to submission email settings page

### PR DESCRIPTION
**Pull request by tomas-stefano**
[Trello card](https://trello.com/c/6olWKyVB/1516-the-sample-pdf-links-arent-working)

## Context

Adding the sample pdf link.

### CODE REVIEW

File: app/views/settings/email/_form.html.erb [hunk 1]
```diff
@@ -40,7 +40,7 @@
       </h3>
     </legend>
 
-    <a href="#" class="govuk-link">See sample PDF</a>
+    <%= link_to 'See sample pdf',asset_url('/sample.pdf'), class: 'govuk-link', target: '_blank' %>
```
Comment (njseeto) [inline app/views/settings/email/_form.html.erb:43]: Just to match the prototype, can we change to 'See sample PDF'?

**Review by njseeto — COMMENTED** [review]
🚀  Just one super small comment, otherwise looks great!

File: app/views/settings/email/_form.html.erb (hunk 1 above)
Comment (tomas-stefano) [reply app/views/settings/email/_form.html.erb]: Ooops! Done!

**Review by njseeto — APPROVED** [review]
🚀

**Pull request merged**